lib/pushmi_pullyu/aip/creator.rb

```diff
@@ -0,0 +1,66 @@
+require 'archive/tar/minitar'
+require 'bagit'
+require 'pushmi_pullyu/aip'
+
+class PushmiPullyu::AIP::Creator
+
+  attr_reader :aip_filename, :aip_directory
+
+  def initialize(noid)
+    @noid = noid
+    @aip_directory = PushmiPullyu::AIP.aip_directory(noid)
+    @aip_filename = PushmiPullyu::AIP.aip_filename(noid)
+    PushmiPullyu::AIP.validate(noid)
+  end
+
+  def run(should_skip_download: false, should_clean_work_directories: true)
+    download_aip unless should_skip_download
```

Comment:
I would probably decouple the download process away from the creator class. You can call this in the CLI run loop.
Something like this:
```ruby
def run_tick_loop
#....
PushmiPullyu::AIP.download(item) if should_skip_download?
aip_filename = PushmiPullyu::AIP.create(item)
#...
```
Then you can remove the arguements for should_skip_download/and the `download_aip` method in here.

This will give something using the  `PushmiPullyu::AIP.download` method as well since it is currently never being used anywhere else.